new/PRS13981Mikael.xml

```diff
@@ -5,7 +5,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title>Zamikāʾel</title>
+                <title>Mikāʾel</title>
```

Comment:
the title should correspond to the name ) it is not the ID

Reply:
Understand. Done.